application/views/view_log/qso.php

```diff
@@ -514,7 +514,7 @@
                             echo ' active';
                         }
                         echo '">';
-                        echo '<img class="d-block w-100" src="' . base_url() . '/assets/qslcard/' . $image->filename .'" alt="QSL picture #'. $i++.'">';
+                        echo '<img class="d-block w-qsl" src="' . base_url() . '/assets/qslcard/' . $image->filename .'" alt="QSL picture #'. $i++.'">';
```

Comment:
This leads to huge pix and many scrollbars:

![Screenshot from 2023-03-07 12-15-14](https://user-images.githubusercontent.com/7112907/223406921-69f13897-1fe5-4780-abe3-7bc4eff20929.png)

Reply:
I thin

> This leads to huge pix and many scrollbars:
> 
This is because the **w-qsl** is absent in the bootstrap.min.css of the theme which you use.

![изображение](https://user-images.githubusercontent.com/16667173/223424172-0775a00c-157e-4474-8584-6ffe7944814f.png)

If image of the QSL is scaled to 100% i got:
![изображение](https://user-images.githubusercontent.com/16667173/223424584-baa64702-eb8b-4f5c-85f7-a1b0de8676ca.png)

Reply:
Well I was testing with the default theme... I guess it should at least work with that theme. Which theme are you using?

Reply:
> Well I was testing with the default theme... I guess it should at least work with that theme. Which theme are you using?
I'm using Cosmo_Wide theme.

Added w-qsl class to CSS for all themes (commit 79bae6f9482696cbe4f3144393389a33efbf0e10). Found a hires QSL card scan image in my log (https://r1blh.cats-home.net/log//assets/qslcard/1538_1673473963.jpg). It looks like all is OK now.

Default Wide theme:
![изображение](https://user-images.githubusercontent.com/16667173/223440069-c16e11ad-f6d0-4408-afb8-1704e1206bb8.png)
![изображение](https://user-images.githubusercontent.com/16667173/223440746-b07be45b-f783-47b5-8a2e-0e1eabd9ac3f.png)

Default theme:
![изображение](https://user-images.githubusercontent.com/16667173/223440278-e77b8c19-78cd-48c5-8800-2debb5b196b9.png)
![изображение](https://user-images.githubusercontent.com/16667173/223440552-c6d86983-45e9-405e-a683-cfb9fff45c85.png)

Reply:
@phl0 could you check this, please?